lib/colorado_lottery.rb

```diff
@@ -0,0 +1,31 @@
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners, :current_contestants
+
+  def initialize
+    @registered_contestants= {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def interested_and_18?(contestant, game)
+    if contestant.game_interests.include?(game.name) && contestant.age > 18
+      true
+    else
+      false
+    end
+  end
+
+  def can_register?(contestant, game)
+    if interested_and_18?(contestant, game) == true || contestant.out_of_state? == false
+      true
+    else
+      false
+    end
+  end
+
+  def register_contestant(contestant, game)
+    @registered_contestants[game.name] = contestant
+  end
+
+  def eligible_contestants(pick_4)
```

Comment:
you'd want this to be `game`, not `pick_4`-- you'd want it to be generic to work for any game.

Reply:
i also shouldn't see any method implementation if the test isn't built out.